cdap-cli/src/main/java/co/cask/cdap/cli/CLIMain.java

```diff
@@ -91,6 +94,14 @@
   public static final Option DEBUG_OPTION = new Option(
     "d", "debug", false, "Print exception stack traces.");
 
+  @VisibleForTesting
+  public static final Option SCRIPT_FILE_OPTION = new Option(
+    "f", "scriptfile", true, "Name of script file to read and execute.  " +
+    "Blank lines and comments starting with '#' are ignored.  " + 
+    "The 'break' keyword terminates the script.  " + 
+    "The 'exit' command terminates the cli, otherwise the user is left " + 
```

Comment:
Maybe have CLI terminate-at-end be the default and then if you do a `break` at the end, it'll leave the user at the cdap-cli command prompt?